### DISCUSSION TITLE
Changing extent of 3D map

### DISCUSSION THREAD

**Discussion by ced00001** [h-a-graham/rayvista#14] [Q&A]
Hello, 

Can you advise me how to change the extent of the map, e.g. if I want a rectangular shape how do I do this?

Many thanks

**Comment by h-a-graham** [comment]
Hey, at the moment, there is only limited supported for requested extents and things will always turn out pretty square. I would be happy to add an argument that took an sf (readable) object and used it's extent to define the area. Is this what you had in mind?

**Reply by ced00001** [reply]
Hi, 

I admit I dont know how that would work but if its not too hard to do (for you or as a user!) it would be great to have that option. Awesome package by the way, it really takes the complexity out of sourcing data for Rayshader and the satellite imagery is high quality which is great.

**Reply by h-a-graham** [reply]
Thanks very much - pleased your finding it useful! I'll add this as an ehancement and try to add it when I get time - quite busy at the moement so possibly won't be for a few weeks. I'll repost here once I've aded this. 
Cheers!

**Reply by h-a-graham** [reply]
Okay, so I've made some updates which now implement this. USe the req_area argument to specify a region of interest. you can supply an sf object or a spatial vector file which can be read by sf such as a .shp file or .gpkg etc. The extent of this object is used to extract the requested region. Hope that makes sense!?
